### ISSUE TITLE
Add Unlicense into the list of licenses

### ISSUE THREAD

**Issue by KOLANICH** [calpaterson/csvbase#92]
# Description

I consider PDDL as too long and prefer Unlicense.

**Comment by calpaterson** [comment]
Perhaps "PDDL" should say "public domain" instead?  And you can link to the specific licence in the README?

**Comment by KOLANICH** [comment]
>Perhaps "PDDL" should say "public domain" instead?

No. PDDL is a public-domain**-like license**, as Unlicense is. There is ain't no such a thing as "public domain-dedicated works" in some states, so just statement "public domain" is void there. And anti-liability clauses are also must have, that's why I never use another public-domain**-like license** called WTFPL, that lacks them (and also itself copyrighted).

2BH I think that the whole set of licenses should be put there. There are datasets of them in JSON format.

**Comment by calpaterson** [comment]
> 2BH I think that the whole set of licenses should be put there. There are datasets of them in JSON format.

That's a good idea, I think that's the way to go.  Perhaps with some culling just to the ones that are suitable for data.
